src/lib/intlFormat.ts

```diff
@@ -88,17 +88,17 @@ export const dateFromParts = (
     return null;
   }
 
-  if (typeof year === 'number') {
-    const date = DateTime.local(year, month, day);
-    if (date.invalidReason || date.invalidExplanation) {
-      return `Invalid Date - ${date.invalidExplanation}`;
+  const date = validateAndFormatInvalidDate(year, month, day, locale);
+  if (date.dateTime.invalidExplanation) {
```

Comment:
Is `invalidReason` no longer a thing?

Reply:
There isn't a need for it. If there is an `invalidExplanation`, we know it will be invalid, so we don't need to also check for `invalidReason`.